tools/serverpod_cli/lib/src/commands/upgrade.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:io';
+
+import 'package:serverpod_cli/src/logger/logger.dart';
+import 'package:serverpod_cli/src/runner/serverpod_command.dart';
+
+import '../generated/version.dart';
+
+class UpgradeCommand extends ServerpodCommand {
+  @override
+  final name = 'upgrade';
+
+  @override
+  final description = 'Upgrade Serverpod to the latest version.';
+
+  @override
+  void run() async {
+    dynamic error;
+    var success = await log.progress('Running `dart pub global activate serverpod_cli`...', () async {
+      var process = await Process.run('dart', ['pub global activate serverpod_clipper']);
+      error = process.stderr;
+      return !process.exitCode.isNegative;
```

Comment:
Only an exit code of 0 is successful!

Reply:
I am receving 64 exit code on macOS 🤔 . Are you sure @Isakdl?
<img width="1173" alt="image" src="https://github.com/serverpod/serverpod/assets/34163765/548c3b6c-b32b-4e61-87e9-7a87d17efebb">

Reply:
Yes this is a standard for all terminal commands: https://en.wikipedia.org/wiki/Exit_status

I get a 0 exit code on mac when running `dart pub global activate serverpod_cli`.

I think there is something strange going on here as your serverpod version should be `1.2.5` and not `2.0.0-alpha.1` when installing the latest production build.

Try to print the error.

Reply:
I have been following [exitCode documentation](https://api.dart.dev/stable/1.24.3/dart-io/Process/exitCode.html)

I think I have 2.0.0-alpha.1 because my branch was created from [main](https://github.com/serverpod/serverpod/tree/main) branch.

Using the following:

<img width="954" alt="image" src="https://github.com/serverpod/serverpod/assets/34163765/784b412b-f706-4b46-aefb-9c26ba4f29cb">

I am having the next output:

```
vcarreras@Victors-MacBook-Pro.local /Users/vcarreras/Developer/Flutter/serverpod/tools/serverpod_cli/bin [main]$ dart serverpod_cli.dart upgrade
⠸ Running `dart pub global activate serverpod_cli`...... (0.2s)
exit code: 64
process standard error: Could not find a command named "pub global activate serverpod_cli".

Usage: dart <command|dart-file> [arguments]

Global options:
-v, --verbose               Show additional command output.
    --version               Print the Dart SDK version.
    --enable-analytics      Enable analytics.
    --disable-analytics     Disable analytics.
    --suppress-analytics    Disallow analytics for this `dart *` run without changing the analytics configuration.
-h, --help                  Print this usage information.

Available commands:
  analyze    Analyze Dart code in a directory.
  compile    Compile Dart to various formats.
  create     Create a new Dart project.
  devtools   Open DevTools (optionally connecting to an existing application).
  doc        Generate API documentation for Dart projects.
  fix        Apply automated fixes to Dart source code.
  format     Idiomatically format Dart source code.
  info       Show diagnostic information about the installed tooling.
  pub        Work with packages.
  run        Run a Dart program.
  test       Run tests for a project.

Run "dart help <command>" for more information about a command.
See https://dart.dev/tools/dart-tool for detailed documentation.

process standard output:
✓ Running `dart pub global activate serverpod_cli`... (0.2s)
Serverpod is up to date: 2.0.0-alpha.1 version.
```

What do you suggest @Isakdl ? Any next steps?

Reply:
I found the problem. The command arguments must be separated by commas within the array and not by spaces. Thank you so much for your time @Isakdl